### PR TITLE
SEEDS

### PR DESCRIPTION
**Pull request by mwilkins74**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,56 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+Game.destroy_all
```
Comment (kmdunn5) [inline db/seeds.rb]: I think we can turn this check into something like `return unless Game.count == 0`. If we run `db:reset` or something like that, it'll destroy the records anyway.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+    name: "Lost Cities",
+    description: "Send expeditions into ancient ruins",
+    category: "Strategy",
+    min_players: 2,
+    max_players: 0,
+    rating: 8
+  }
+]
+
+game_data.each do |game|
```
Comment (kmdunn5) [inline db/seeds.rb]: we can do a `Game.find_by(name: game.name)` here to make sure we don't duplicate if you end up removing the destroy above.

**Review by kmdunn5 — APPROVED** [review]
Your choice on the comments below. Merge if you feel like you always want to destroy on seeding, change if you don't want to do that.

File: db/seeds.rb (hunk 1 above)
Comment (mwilkins74) [reply db/seeds.rb]: Yeah, that makes way more sense.

File: db/seeds.rb (hunk 2 above)
Comment (mwilkins74) [reply db/seeds.rb]: Way better idea!

**Pull request merged**